src/test/java/de/mediathekview/mserver/testhelper/FileReader.java

```diff
@@ -1,24 +1,41 @@
 package de.mediathekview.mserver.testhelper;
 
+import static org.junit.Assert.fail;
+
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import static org.junit.Assert.fail;
 
 public class FileReader {
-    private FileReader() {}
-    
-    public static String readFile(String filePath) {
-        try {
-            URI u = FileReader.class.getResource(filePath).toURI();
-            Path path = Paths.get(u);
-            return new String(Files.readAllBytes(path), "UTF-8");
-        } catch(IOException | URISyntaxException ex) {
-            fail("Exception reading file " + filePath + ": " + ex.getMessage());
-        }
-        return null;
+
+  private FileReader() {
+  }
+
+  public static String readFile(String filePath) {
+    try {
+      Path path = getPath(filePath);
+      return new String(Files.readAllBytes(path), "UTF-8");
+    } catch (IOException ex) {
+      fail("Exception reading file " + filePath + ": " + ex.getMessage());
+    }
+    return null;
+  }
+
+  public static Path getPath(String filePath) {
+
+    if (!filePath.startsWith("/")) {
+      filePath = "/" + filePath;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this hard-coded path-delimiter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)